src/constants/stats.ts

```diff
@@ -69,3 +69,40 @@ export const Stats: StatMapping = {
   BetAndWinDrawOdds: "bwd",
   BetAndWinAwayOdds: "bwa",
 };
+
+// create statsArray with the format: {label: 'League Division', value: 'div'}
+export const FilteringStatsArray = [
+  { label: "Home Team Average Age", value: "homeAvgAge" },
```

Comment:
Instead of repeating the value fields here, can you reuse them from the Stats object above?

Like:
```
{ label: "Away Team Average Age", value: Stats.AwayTeamAverageAge },
```